scss/navbar.scss

```diff
@@ -1,7 +1,7 @@
 .navbar {
   align-items: center;
   background-color: $white;
-  box-shadow: 6px 6px 5px #D3D0CD;
+  box-shadow: 6px 6px 5px $box-shadow;
```

Comment:
Change the name for something more descriptive, i recommend you create a variable called `$black: #000` and here use it as `rgba($black, 0.4)`